CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing
+
+## How docs.ruby-lang.org/en Works
+
+### Overview
+
+The English Ruby documentation is automatically generated and updated through a pipeline involving GitHub Actions, AWS S3, and systemd services. Here's how it all comes together:
+
+1. **Documentation Generation and Upload**
+
+   - A scheduled [GitHub Action](https://github.com/ruby/actions/blob/master/.github/workflows/docs.yml) generates the documentation for all [actively maintained Ruby versions](https://www.ruby-lang.org/en/downloads/branches/) plus the master branch. Each documentation set is generated using the version of RDoc bundled with that specific Ruby version.
+     - *Example:* For [Ruby 3.3](https://docs.ruby-lang.org/en/3.3/), the documentation is generated with RDoc 6.6.3.1.
+   - The generated docs are uploaded as `ruby-docs-en-<version>.tar.xz` to the `ftp.r-l.o/pub/ruby/doc/` S3 bucket.
```

Comment:
It took me a moment to understand the r-l.o abbreviation. Is this actually the bucket name or a shortcut in this doc?